deps/rabbitmq_amqp1_0/src/rabbit_amqp1_0_reader.erl

```diff
@@ -66,7 +67,8 @@ unpack_from_0_9_1({Parent, Sock,RecvLen, PendingRecv,
                                     frame_max      = ?FRAME_MIN_SIZE,
                                     auth_mechanism = none,
                                     auth_state     = none},
-        proxy_socket = ProxySocket}.
+        proxy_socket = ProxySocket,
+        tracked_channels = []}.
```

Comment:
Use a map instead of a proplist. Better in so many ways.

Reply:
See my previous comment. Maybe it was a misunderstanding.